web/pandas/pdeps/0008-inplace-methods-in-pandas.md

```diff
@@ -348,21 +347,22 @@ DataFrames. Therefore, we decided to keep the `inplace` keyword for this small s
 ### Standardize on the `copy` keyword instead of `inplace`
 
 It may seem more natural to standardize on the `copy` keyword instead of the `inplace` keyword, since the `copy`
-keyword already returns a new object instead of None (enabling method chaining) when it is set to `True`.
+keyword already returns a new object instead of None (enabling method chaining) and avoids a coopy when it is set to `False`.
 
 However, the `copy` keyword is not supported in any of the values-mutating methods listed in Group 2 above
 unlike `inplace`, so semantics of future inplace mutation of values align better with the current behavior of
 the `inplace` keyword, than with the current behavior of the `copy` keyword.
 
 Furthermore, with the Copy-on-Write proposal, the `copy` keyword also has become superfluous. With Copy-on-Write
 enabled, methods that return a new pandas object will always try to avoid a copy whenever possible, regardless of
-a `copy=False` keyword. Thus, the general proposal is to actually remove the `copy` keyword from the methods where it is
-currently used.
-
-Currently, for methods where it is supported, when the `copy` keyword is `False`, a new pandas object (same
-as `copy=True`) is returned as the result of a method call, with the values backing the object being shared when
-possible. With the proposed inplace behavior, current behavior of `copy=False` would return a new pandas object with
-identical values as the original object(that was modified inplace), which may be confusing for users, and lead to
+a `copy=False` keyword. Thus, the Copy-on-Write PDEP proposes to actually remove the `copy` keyword from the methods
+where it is currently used (so it would be strange to add this as a new keyword to the Group 2 methods).
+
+Currently, when using `copy=False` in methods where it is supported, a new pandas object is returned as the result
+of a method call (same as with `copy=True`), but with the values backing this object being shared with the calling
+object when possible (but the calling object is never modified). With the proposed inplace behavior for Group 2 methods,
```

Comment:
Maybe we can reword the second part of this sentence (after the comma), to
``sharing underlying values with the calling object if the method does not mutate those values``.